sdks/python/setup.py

```diff
@@ -125,15 +125,15 @@ def get_version():
     'pytz>=2018.3',
     # [BEAM-5628] Beam VCF IO is not supported in Python 3.
     'pyvcf>=0.6.8,<0.7.0; python_version < "3.0"',
-    'pyyaml>=3.12,<4.0.0',
+    'pyyaml>=3.13,<4.0.0',
```

Comment:
Why do we need this change?

Reply:
oh, this was by mistake, 3.13 is in the range, so no need to change.